scrapers_next/pa/committees.py

```diff
@@ -0,0 +1,171 @@
+from spatula import CSS, HtmlPage, HtmlListPage, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+class HouseCommitteeDetail(HtmlPage):
+    example_source = "https://www.legis.state.pa.us/cfdocs/CteeInfo/index.cfm?Code=32&CteeBody=H&SessYear=2021"
+    example_name = "Aging & Older Adult Services"
+    example_input = ScrapeCommittee(
+        name=example_name, classification="committee", chamber="lower"
+    )
+
+    def process_page(self):
+        com = self.input
+        try:
+            # This section has the chair memebers the regular, democratic and minority and the roles
+            # main chair
+            Chair_Member = (
+                CSS("div.MemberInfoList-MemberWrapper.ChairWrapper div.ChairNameText a")
+                .match(self.root)[0]
+                .text.strip()
+            )
+            # main chair role
+            Chair_Member_role = (
+                CSS(
+                    "div.MemberInfoList-MemberWrapper.ChairWrapper div.ChairNameText div"
+                )
+                .match(self.root)[0]
+                .text.strip()
+            )
+            com.add_member(Chair_Member, Chair_Member_role)
+            # Democratic Chair member and or the minority chair member
+            Demo_Chair_Member = (
+                CSS("div.MemberInfoList-MemberWrapper.ChairWrapper div.ChairNameText a")
+                .match(self.root)[1]
+                .text.strip()
+            )
+            # Democratic Chair member and or the minority chair member role
+            Demo_Chair_Member_role = (
+                CSS(
+                    "div.MemberInfoList-MemberWrapper.ChairWrapper div.ChairNameText div"
+                )
+                .match(self.root)[1]
+                .text.strip()
+            )
```

Comment:
I notice in this section you are repeating (most of) a selector string four times. This is a risky code pattern because a) it is not clear to a later developer if these are actually the same four strings or not, and b) someone could try to fix a bug and only change one or two of the strings, not realizing that all four need to be changed. This needs to be refactored (changed) to use a different pattern. We talked about this previously.

One option is to use [f-strings](https://realpython.com/python-f-strings/) so that you can do 

``` 
chair_details_selector = "div.MemberInfoList-MemberWrapper.ChairWrapper div.ChairNameText"
chair_member = CSS(f"{chair_details_selector} a")
```

But an even better refactor would be to first select the container for each chair, then loop through and perform operations on each sub-selection to get individual pieces of data (rather than a list of four seemingly unrelated selections):

```
In [7]: chair_elements = CSS(".MemberInfoList-MemberWrapper.ChairWrapper").match(root)
for chair_element in chair_elements:
In [14]: for chair_element in chair_elements:
    ...:     member_name = CSS("a").match_one(chair_element).text.strip()
             ...etc.
             com.add_member(member_name, member_role)
```